packages/mobile/src/screens/chat-screen/ChatMessageListItem.tsx

```diff
@@ -70,30 +66,24 @@ const useStyles = makeStyles(({ spacing, palette, typography }) => ({
   link: {
     textDecorationLine: 'underline'
   },
+  shadow: {
+    shadowColor: 'black',
+    shadowOpacity: 0.2,
+    shadowRadius: 5,
+    shadowOffset: { width: 0, height: 3 }
+  },
   tail: {
     display: 'flex',
     position: 'absolute',
-    bottom: 47
+    bottom: -0.4
```

Comment:
Yikes. Is this going to cause anything weird on different devices? I feel like a sub-pixel offset should be unnecessary.

Reply:
+1 on the yikes. I did check on actual device and it worked. There's a tiny offset at the bottom of the svg, would have to ask design to update it.